mdpow/tests/test_solv_shell.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+from . import tempdir as td
+
+import py.path
+
+import pybol
+import pytest
+
+from numpy.testing import assert_almost_equal
+from scipy.stats import variation
+
+from ..analysis.ensemble import Ensemble, EnsembleAnalysis, EnsembleAtomGroup
+
+from ..analysis.solvation import SolvationAnalysis
+
+from pkg_resources import resource_filename
+
+RESOURCES = py.path.local(resource_filename(__name__, 'testing_resources'))
+MANIFEST = RESOURCES.join("manifest.yml")
+
+
+class TestSolvShell(object):
+    mean = 2654.0
```

Comment:
remove and replace with a parametrized test (see below)

Reply:
done